### DISCUSSION TITLE
@e[] selectors

### DISCUSSION THREAD

**Discussion by meepstertron** [EssentialsX/Essentials#4559] [Ideas and Feature Suggestions] [answered]
hello i just made a game w comand blocks and essentialsX just broke it beacause you cant do @e[type=snowgolem]

**Comment by Octol1ttle** [comment, marked as answer]
Open your `commands.yml` file in the **root folder** of your server, replace this:
`command-block-overrides: []`
With this:
```
command-block-overrides:
    - "*"
```

**Reply by meepstertron** [reply]
thanks
:)
